metomi/isodatetime/tests/test_00.py

```diff
@@ -1084,9 +1093,6 @@ def test_timepoint_dumper(self):
                     num_expanded_year_digits=num_expanded_year_digits)
                 self.assertRaises(ctrl_exception, dumper.dump,
                                   ctrl_timepoint, format_)
-        value_error_timepoint = data.TimePoint(minute_of_hour=10)
-        value_error_timepoint._minute_of_hour = "1O"
-        self.assertRaises(ValueError, dumper.dump, value_error_timepoint, "%M")
```

Comment:
Behaviour change?

Reply:
Now that TimePoints are "immutable", we shouldn't have to worry about mutating a property to an invalid value